### PR TITLE
CLI Print Pretty

### PR DESCRIPTION
**Pull request by mostafakhudair**
Remove unless before/after newLine()
Fix Windows cmd colors
Remove useless methods

Here's some examples:
https://i.ibb.co/r3JnQPs/Annotation-2020-07-31-210349.png
https://i.ibb.co/SnwwdJV/Annotation-2020-07-31-213208.png

### CODE REVIEW

File: system/CLI/Console.php [hunk 1]
```diff
@@ -93,14 +93,7 @@ public function run(bool $useSafeOutput = false)
 	 */
 	public function showHeader()
 	{
-		CLI::newLine();
-
-		CLI::write(CLI::color('CodeIgniter CLI Tool', 'green')
-				. ' - Version ' . CodeIgniter::CI_VERSION
-				. sprintf(' - Server Time: %s UTC%s', date('Y-m-d H:i:sa'), date('P')));
-
+		CLI::write('CodeIgniter v' . CodeIgniter::CI_VERSION . ' Command Line Tool - Server-Time: ' . date('Y-m-d H:i:s'), 'light_blue');
```
Comment (paulbalandan) [inline system/CLI/Console.php:96]: Why remove the UTC offset? I just made a PR days ago to include that, though.

File: system/Commands/ListCommands.php [hunk 2]
```diff
@@ -115,82 +107,43 @@ public function run(array $params)
...
-
-				$out = CLI::color($title, 'yellow');
-
-				if (isset($item['description']))
-				{
-					$out .= CLI::wrap($item['description'], 125, strlen($title));
-				}
+				CLI::write(
+					CLI::color($this->setPad($command['name'], $length, 2, 2), 'green')
+					 . $command['description']
```
Comment (paulbalandan) [inline system/Commands/ListCommands.php:137]: If we'll remove the wrap here then long descriptions would span across the terminal width and overflow.

File: system/CLI/Commands.php [hunk 3]
```diff
@@ -173,20 +175,21 @@ public function discoverCommands()
...
 
 				// Store it!
-				if ($class->group !== null)
+				if (! is_null($class->group))
 				{
 					$this->commands[$class->name] = [
 						'class'       => $className,
 						'file'        => $file,
 						'group'       => $class->group,
+						'name'	      => $class->name,
```
Comment (paulbalandan) [inline system/CLI/Commands.php:184]: Is there a benefit of adding this line? As I see it, the command's name is already used as the array key and can be accessed later by foreach using `foreach ($commands as $name => $command)`, which is the current way.

File: system/Commands/ListCommands.php [hunk 4]
```diff
@@ -73,7 +73,7 @@ class ListCommands extends BaseCommand
 	 *
 	 * @var string
 	 */
-	protected $description = 'Lists the available commands.';
+	protected $description = 'List of available commands.';
```
Comment (paulbalandan) [inline system/Commands/ListCommands.php:76]: I think the former is more apt as it describes the command's function, i.e. what it is doing.

**Comment by michalsn** [comment]
Changes look good, but I would like to hear from @paulbalandan first. He made a lot of quality changes to the `CLI` lately, so he knows this class better than me right now :)

**Comment by michalsn** [comment]
Thank you both!

**Pull request merged**